### ISSUE TITLE
Dataset at ftp:/podaac-ftp.jpl.nasa.gov/allData/insitu

### ISSUE THREAD

**Issue by nickrsan** [climate-mirror/datasets#193]
ftp:/podaac-ftp.jpl.nasa.gov/allData/insitu.

Suggested in a large email containing many urls

**Comment by rozap** [comment]
downloading

**Comment by donbright** [comment]
Mirror: offline, contact me if data is needed
Size: 2.3G
Hashdeep: offline, contact me if data is needed

----
redownloaded 2017-2-19 due to hard drive failure

**Comment by bkirkbri** [comment]
@rozap Thanks! Please update when your mirror is complete.
@donbright Thank you
@adinbied Your mirror at https://github.com/climate-mirror/datasets/issues/244#issuecomment-275288586 matches except for these files:

```
allData/insitu/L2/spurs1/README.txt
allData/insitu/L2/spurs1/README_Changelog_DrifterDataset.txt
allData/insitu/L2/spurs1/sw/R/README_R_netCDF.txt
allData/insitu/L2/spurs1/docs/MeetingMaterials/README_MeetingMaterials.txt
allData/insitu/L2/spurs1/README_KnownIssues.txt
```

I suspect these files changed between the two mirrors being made.

**Comment by adinbied** [comment]
@bkirkbri Yup, i'm pretty sure that's what happened. Should I bother to re-mirror?

**Comment by gabefair** [comment]
I have a copy here: https://www.dropbox.com/sh/l1hkna5hng74zs1/AAAyBSpNnGMwbACfefsZdVdpa?dl=0

**Comment by bkirkbri** [comment]
@adinbied No need. Thank you.

**Comment by gabefair** [comment]
1 vote to close
